### PR TITLE
Add Econsave mini market branch

### PR DESCRIPTION
**Pull request by MASangys**
see https://econsave.com.my/all-stores/

### CODE REVIEW

File: data/brands/shop/variety_store.json [hunk 1]
```diff
@@ -517,6 +517,19 @@
...
       }
     },
+    {
+      "displayName": "Pasar Ekonomi Econsave",
+      "id": "pasarekonomieconsave-c7d002",
+      "locationSet": {"include": ["my"]},
+      "tags": {
+        "brand": "Pasar Ekonomi Econsave",
+        "brand:zh": "宜康省经济市场",
+        "brand:wikidata": "Q61776108",
```
Comment (bhousel) [inline data/brands/shop/variety_store.json:527]: https://www.wikidata.org/wiki/Q61776108 looks wrong - it goes to something unrelated?

**Review by bhousel — CHANGES_REQUESTED** [review]
Q61776108 wrong

**Comment by MASangys** [comment]
Q61776608

**Comment by MASangys** [comment]
Hello, any update?

**Review by bhousel — APPROVED** [review]

**Comment by bhousel** [comment]
> Hello, any update?

Thanks, looks fine now.. We had a holiday here and I wasn't paying much attention for the past few days.

**Pull request merged**